src/dialects/abstract/query-generator.js

```diff
@@ -712,15 +668,9 @@ export class AbstractQueryGenerator {
   }
 
   addConstraintQuery(tableName, options) {
-    if (typeof tableName === 'string') {
-      tableName = this.quoteIdentifiers(tableName);
```

Comment:
something else that's been removed:

You used to be able to call `addConstraintQuery('schema.table')` and it'd split the string in two.

The thing is that `.` is a valid character in table names, so now functions like this one produce `"schema.table"` instead of `"schema"."table"`. If you want to specify a schema, pass `addConstraintQuery({ schema, table })` instead of a string